### PR TITLE
virttest.utils_misc: add some steps about partition

### PR DESCRIPTION
**Pull request by maxujun**

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -2755,14 +2755,25 @@ def format_linux_disk(session, did, mountpoint, size, fstype="ext3"):
...
             mkpart_cmd = "parted -s %s  mkpart primary 0 %s" % (kname, size)
             session.cmd(mkpart_cmd)
+            time.sleep(5)
+            logging.info("Try to get the partitions of '%s'" % kname)
+            get_part_cmd = "readlink -f /dev/disk/by-id/*|grep %s\\\w" % kname
+            parts = session.cmd(get_part_cmd).splitlines()
+            logging.info("The partitions are '%s'" % parts)
             logging.info("Format partition to '%s'" % fstype)
-            format_cmd = "yes|mkfs -t %s %s" % (kname, fstype)
+            format_cmd = "yes|mkfs  %s -t %s" % (parts[-1], fstype)
```
Comment (PandaWei) [inline virttest/utils_misc.py]: Here you use parts[-1] instead of kname.

File: virttest/utils_misc.py [hunk 2]
```diff
@@ -2755,14 +2755,25 @@ def format_linux_disk(session, did, mountpoint, size, fstype="ext3"):
...
             session.cmd(format_cmd)
             logging.info("Mount the disk to '%s'" % mountpoint)
-            mount_cmd = "mount -t %s %s %s" % (fstype, kname, mountpoint)
+            mount_cmd = "mount -t %s %s1 %s" % (fstype, kname, mountpoint)
```
Comment (PandaWei) [inline virttest/utils_misc.py]: Here use kname.
Why？ It's better to  keep consistence.

Comment (PandaWei) [inline virttest/utils_misc.py]: And sorry, can you explain to me why use "%s1"?
Thanks

**Comment by maxujun** [comment]
@PandaWei hi, I forgot  to update the parameter of mount_cmd,and have updated it now.thanks for  reminding.

**Pull request closed without merging**